File: Python/Eliminacion_al_inicio.py
arreglo = [10, 20, 30, 40]
n = 4

def el_inicio():
    global n
    for i in range(0, n - 1):
        arreglo[i] = arreglo[i + 1]
    n -= 1

el_inicio()
print(arreglo[:n])

File: Java/Eliminacion_al_inicio.java
public class Eliminacion_al_inicio {
    public static void main(String[] args) {
        int[] arreglo = {10, 20, 30, 40};
        int n = 4;

        for (int i = 0; i < n - 1; i++) {
            arreglo[i] = arreglo[i + 1];
        }
        n--;

        for (int i = 0; i < n; i++)
            System.out.print(arreglo[i] + " ");
    }
} 
